### PR TITLE
Better error handling for mma page

### PR DESCRIPTION
**Pull request by AWare**
## What does this change?
Currently, if the manage-my-account page fails to load the users details from the members-data-api, it just spins forever. 

This will make it show a nice error message, instructing the user to get in touch.

## What is the value of this and can you measure success?

Fewer complaints.

## Tested in CODE?

cc @paulbrown1982 for OK on messaging

### CODE REVIEW

**Review by paulbrown1982 — APPROVED** [review]
👍  LGTM

File: static/src/javascripts-legacy/projects/membership/membership-tab.js [hunk 1]
```diff
@@ -54,10 +56,14 @@ define([
...
                 populateUserDetails(json);
-            } else {
+                } else {
                 hideLoader();
                 displayMembershipUpSell();
             }
+        }).catch(function (){
+            hideLoader();
+            displayErrorMessage();
+            //TODO: HANDLE THIS ERROR
```
Comment (paulbrown1982) [inline static/src/javascripts-legacy/projects/membership/membership-tab.js]: How will this be handled?

File: static/src/javascripts-legacy/projects/membership/membership-tab.js [hunk 2]
```diff
@@ -54,10 +56,14 @@ define([
             if (json && json.subscription) {
                 hideLoader();
                 populateUserDetails(json);
-            } else {
+                } else {
```
Comment (paulbrown1982) [inline static/src/javascripts-legacy/projects/membership/membership-tab.js]: While you're updating the comment, there's this bit of whitespace.

**Review by SiAdcock — APPROVED** [review]
Tidy 👏

**Pull request merged**